src/stylesheets/components/_dialog.scss

```diff
@@ -29,6 +29,11 @@
   max-height: 65vh;
   overflow: auto;
   -webkit-overflow-scrolling: touch;
+  .text-base{
```

Comment:
text-base is just a color utility, we shouldn't use this as the selector for centering content.

I would suggest adding a variant class named sds-dialog-content--centered to the sds-dialog-content and use that for the selector.

Reply:
sure thing, updated